src/rules/unit-no-unknown/index.js

```diff
@@ -0,0 +1,60 @@
+import { isString } from "lodash"
+import valueParser from "postcss-value-parser"
+
+import {
+  optionsHaveIgnored,
```

Comment:
Minor issue, these tend to be alphabetically ordered. It makes scanning through rules a little easier.